testsuite/OMSimulator/snapshot.py

```diff
@@ -106,6 +106,15 @@
 ## 			</ssv:Parameters>
 ## 		</ssv:ParameterSet>
 ## 	</oms:ssv_file>
+## 	<oms:signalFilter_file name="resources/signalFilter.xml">
+## 		<oms:SignalFilter version="1.0">
+## 			<oms:Variable name="snapshot.root.add.u1" type="real" kind="input" />
+## 			<oms:Variable name="snapshot.root.add.u2" type="real" kind="input" />
+## 			<oms:Variable name="snapshot.root.add.y" type="real" kind="output" />
+## 			<oms:Variable name="snapshot.root.add.k1" type="real" kind="parameter" />
+## 			<oms:Variable name="snapshot.root.add.k2" type="real" kind="parameter" />
```

Comment:
SSP and FMI use `Real` instead for `real`, don't they?

Reply:
fixed with this https://github.com/OpenModelica/OMSimulator/pull/915/commits/ae8174960b62f59c22f92e87c336328bd0194111